docs/cloud-native-security/cspm-get-started-azure.asciidoc

```diff
@@ -22,7 +22,7 @@ This page explains how to get started monitoring the security posture of your cl
 [[cspm-setup-azure]]
 == Set up CSPM for Azure
 
-To set up CSPM for Azure, first add the CSPM integration, then enable cloud account access.
+You can set up CSPM for Azure either by by enroling an Azure organization containing multiple subscriptions, or by enroling a single subscription. Either way, first you will add the CSPM integration, then enable cloud account access.
```

Comment:
Let's say "Azure organization (root management group)" instead of just Azure orgnization.

Reply:
or just "Azure organization (management group)" since non-root management groups work as well